libsolidity/formal/SMTChecker.cpp

```diff
@@ -623,6 +625,29 @@ void SMTChecker::endVisit(FunctionCall const& _funCall)
 	case FunctionType::Kind::MulMod:
 		abstractFunctionCall(_funCall);
 		break;
+	case FunctionType::Kind::Send:
+	case FunctionType::Kind::Transfer:
+	{
+		auto const& memberAccess = dynamic_cast<MemberAccess const&>(_funCall.expression());
+		auto const& address = memberAccess.expression();
+		auto const& value = args.at(0);
+		solAssert(value, "");
+
+		addOverflowTarget(
+			OverflowTarget::Type::Overflow,
+			make_shared<IntegerType>(256),
+			m_context.balance(expr(address)) + expr(*value),
+			_funCall.location()
+		);
```

Comment:
Does it make sense to test transfer for overflow? In practice this will never happen...